site/content/3.11/release-notes/version-3.11/api-changes-in-3-11.md

```diff
@@ -834,7 +834,20 @@ pregel.removeHistory();
 See [Distributed Iterative Graph Processing (Pregel)](../../data-science/pregel/_index.md#get-persisted-execution-statistics)
 for details.
 
-### Deprecations
+### `collection.iterate()` deprecated
 
 The `collection.iterate()` method is deprecated from v3.11.0 onwards and will be
 removed in a future version.
+
+### `@arangodb/request` certificate validation
+
+<small>Introduced in: v3.11.11</small>
+
+The `@arangodb/request` module now supports two additional options for making
+HTTP requests:
```

Comment:
probably better: for making `HTTPS/TLS` requests?

Reply:
Good point. I think "HTTPS" is nicer than "HTTP over TLS/SSL"